tests/test_fv3run.py

```diff
@@ -265,3 +279,83 @@ def test_get_credentials_args(keyfile, expected_docker_args, expected_bind_mount
     )
     assert docker_args == expected_docker_args
     assert bind_mount_args == expected_bind_mount_args
+
+
+@pytest.mark.parametrize(
+    "config_dict, local_paths",
+    [
+        [
+            {
+                "experiment_name": "default_experiment",
+                "initial_conditions": "default",
+                "forcing": "gs://bucket/forcing/default",
+                "diag_table": "gs://bucket/diag/default",
+                "data_table": "gs://bucket/data_table/default",
+            },
+            [],
+        ],
+        [
+            {
+                "experiment_name": "default_experiment",
+                "initial_conditions": "/local/initial/conditions",
+                "forcing": "gs://bucket/forcing/default",
+                "diag_table": "gs://bucket/diag/default",
+                "data_table": "gs://bucket/data_table/default",
+            },
+            ["/local/initial/conditions"],
+        ],
+        [
+            {
+                "experiment_name": "default_experiment",
+                "initial_conditions": "/local/initial/conditions",
+                "forcing": "/local/forcing/default",
+                "diag_table": "/local/diag/default",
+                "data_table": "gs://bucket/data_table/default",
+            },
+            [
+                "/local/initial/conditions",
+                "/local/forcing/default",
+                "/local/diag/default",
+            ],
+        ],
+        [
+            {
+                "experiment_name": "default_experiment",
+                "initial_conditions": "gs://bucket/initial_conditions/default",
+                "forcing": "gs://bucket/forcing/default",
+                "diag_table": [
```

Comment:
This test is a bit confusing, since in practice `diag_table` cannot be specified as an asset dict. Perhaps this was meant to be the `patch_files` entry? Either way, a config dict that uses the `patch_files` option should be tested.

Reply:
The docs state that initial_conditions, forcing or patch_files can use the asset representation.

We could update the API, but I suggest we also address this as part of #46.